Formula/mkvtomp4.rb

```diff
@@ -8,6 +8,11 @@ class Mkvtomp4 < Formula
   license "MIT"
   revision 2
 
+  livecheck do
+    url :stable
+    regex(/^mkvtomp4-v?(\d+(?:\.\d+)+?)$/i)
```

Comment:
```suggestion
    regex(/^mkvtomp4[._-]v?(\d+(?:\.\d+)+)$/i)
```

We typically use `[._-]` between the software name and version, since it's a little more flexible.

I've replaced the version part of the regex with the standard regex for versions like `1.2`, `1.2.3`, etc., as we could run into problems in the future if the repeating part is made non-greedy (i.e., `(?:\.\d+)+?` instead of `(?:\.\d+)+`).

Reply:
noted, much better. 👍